packages/react-components/react-menu/src/components/MenuTrigger/useMenuTrigger.ts

```diff
@@ -129,19 +123,22 @@ export const useMenuTrigger_unstable = (props: MenuTriggerProps): MenuTriggerSta
 
   return {
     isSubmenu,
-    children: applyTriggerPropsToChildren<MenuTriggerChildProps>(children, {
-      'aria-haspopup': 'menu',
-      'aria-expanded': !open && !isSubmenu ? undefined : open,
-      id: triggerId,
-      ...child?.props,
-      ref: useMergedRefs(triggerRef, child?.ref),
-      onClick: useEventCallback(mergeCallbacks(child?.props?.onClick, onClick)),
-      onMouseEnter: useEventCallback(mergeCallbacks(child?.props?.onMouseEnter, onMouseEnter)),
-      onMouseLeave: useEventCallback(mergeCallbacks(child?.props?.onMouseLeave, onMouseLeave)),
-      onKeyDown: useEventCallback(mergeCallbacks(child?.props?.onKeyDown, onKeyDown)),
-      onContextMenu: useEventCallback(mergeCallbacks(child?.props?.onContextMenu, onContextMenu)),
-      onMouseMove: useEventCallback(mergeCallbacks(child?.props?.onMouseMove, onMouseMove)),
-    }),
+    children: applyTriggerPropsToChildren<MenuTriggerChildProps>(
```

Comment:
if a user previously used a context menu
```tsx
<Menu openOnContext>
  <MenuTrigger>
     <div>content</div>
  </MenuTrigger>
</Menu>
```

is that `div` now going to have `role="button` applied to it ?

Reply:
Yes... unless specified otherwise `useARIAButton` adds `role=button` to every non `button` element